tensorbay/client/job.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""Basic structures of asynchronous jobs."""
+
+from typing import Any, Dict, Optional
+
+
+class Job:  # pylint: disable=too-many-instance-attributes
+    """This class defines :class:`Job`.
+
+    Arguments:
+        title: Title of the Job.
+        job_id: ID of the Job.
+        job_type: Type of the Job.
+        creator: The creator of the Job.
+        arguments: Arguments of the Job.
+        created_at: The time when the Job is created.
+        started_at: The time when the Job is started.
+        limit_retry: The limit retry times of the Job.
+        finished_at: The time when the Job is finished.
+        description: The description of the Job.
+
+    """
+
+    def __init__(  # pylint: disable=too-many-arguments
+        self,
+        title: str,
+        job_id: str,
+        job_type: str,
+        creator: str,
+        arguments: Dict[str, Any],
+        created_at: str,
+        started_at: str,
+        limit_retry: int,
+        finished_at: Optional[str] = None,
+        description: Optional[str] = "",
+    ) -> None:
+        self.title = title
+        self.job_id = job_id
+        self.job_type = job_type
+        self.creator = creator
+        self.arguments = arguments
+        self.created_at = created_at
+        self.started_at = started_at
+        self.limit_retry = limit_retry
+        self.finished_at = finished_at
+        self.description = description
+
+    def get_status(self) -> Dict[str, Any]:
```

Comment:
Is it necessary to illustrate the format the return dict?
Same in the `SquashAndMergeJob.get_result`.